TurkishFilter/sections/css_extended.txt

```diff
@@ -17,6 +17,8 @@ geyvemedya.com#?##sideMenu > .widget:has(> .kutu > .textwidget:empty)
 geyvemedya.com#?##sideMenu > .widget:has(> .kutu > .textwidget a[target="_new"]:not([href="https://geyvemedya.com"]))
 geyvemedya.com#?##sideMenu > .widget:has(> .kutu > .textwidget > p > script:contains(reklam))
 mayonez.net##div[class$="tside"] .block:has(>.btl > span:contains(SPONSOR))
+fiisports11.com#?#.pbutton:has(> a[target="_blank"])
```

Comment:
better add with `a[href^="https://fiisports11.com/out?p="]` in extended css part + `a[href^="https://fiisports11.com/out?p="]`

Reply:
changed ext-css rules to` a[href^="https://fiisports11.com/out?p="]`